bin/cron/geocode_hoc_activity

```diff
@@ -7,31 +7,30 @@ BATCH_SIZE = 4000
 THREAD_COUNT = 2
 
 def main
-  threads = [].tap do |threads|
-    THREAD_COUNT.times do |segment|
-      threads << Thread.new do
-        counter_key = "hoc_activity.last_id-#{segment}"
-        at = Properties.get(counter_key).to_i
-        DB.fetch(
-          "SELECT id,started_ip,pixel_started_ip,pixel_finished_ip,finished_ip FROM hoc_activity WHERE id > #{at} AND id < #{at + BATCH_SIZE} ORDER BY id ASC"
-        ).each do |row|
-          next if (row[:id] % THREAD_COUNT) != segment
+  threads = []
+  THREAD_COUNT.times do |segment|
```

Comment:
Looks like in this case the return value isn't important and isn't used. We just want to execute the block `THREAD_COUNT` times. @ashercodeorg please confirm.